core/pom.xml

```diff
@@ -46,6 +50,29 @@
             <version>${forestflow-latest.version}</version>
         </dependency>
 
+        <!-- https://doc.akka.io/docs/alpakka/current/s3.html#artifacts -->
+        <dependency>
+            <groupId>com.lightbend.akka</groupId>
+            <artifactId>akka-stream-alpakka-s3_${scala.binary.version}</artifactId>
```

Comment:
See previous comment. This should be picked up from parent pom property scala.compat.version. The same applies wherever this is used.